frontend/src/layouts/index.jsx

```diff
@@ -46,68 +46,81 @@ const Layout = ({
     theme = 'light',
     toggleTheme,
     language = getCurrentLanguage(),
-    toggleLanguage
+    toggleLanguage,
+    pathContext
 }) => {
     const AddExtraProps = Component => {
         return <Component.type {...Component.props} language={language} />;
     };
     const newComponent = AddExtraProps(children);
-
+    const vr = pathContext.vr || false;
     return (
-        <Provider store={store}>
-            <div
-                style={{
-                    minHeight: '100%',
-                    display: 'flex',
-                    flexDirection: 'column'
-                }}
-            >
-                <Helmet
-                    title="IEEE uOttawa Student Branch"
-                    link={[
-                        {
-                            rel: 'shortcut icon',
-                            type: 'image/png',
-                            href: `${favicon}`
-                        }
-                    ]}
-                >
-                    <meta
-                        property="og:image"
-                        content={`https://ieeeuottawa.ca${logo}`}
-                    />
-                    <meta
-                        property="og:title"
-                        content="IEEE uOttawa Student Branch"
-                    />
-                    <meta
-                        property="og:description"
-                        content="The IEEE uOttawa Student Branch is the official student branch for the University of Ottawa and the official Sub-Association for ELG/CEG/SEG under the ESS. The University of Ottawa’s IEEE Student Branch was established to provide professional services to improve each student’s experience on campus. This includes accommodating students with access to up-to-date equipment, internet access, textbooks and a quiet work environment."
-                    />
-                    <meta property="og:url" content="https://ieeeuottawa.ca/" />
-                    <meta name="twitter:card" content="summary_large_image" />
-                    <meta name="twitter:site" content="@ieeeuottawa" />
-                </Helmet>
-                <Header
-                    theme={theme}
-                    toggleTheme={toggleTheme}
-                    language={language}
-                    toggleLanguage={toggleLanguage}
-                />
-                <div
-                    style={{
-                        margin: '1em auto 0',
-                        paddingTop: '0',
-                        flex: '1 0 auto',
-                        width: '100%',
-                        minHeight: 'calc(100vh - 386px)'
-                    }}
-                >
-                    {newComponent}
-                </div>
-                <Footer language={language} />
-            </div>
-        </Provider>
+        <div>
+            {vr ? (
+                <div>{newComponent}</div>
```

Comment:
should probably keep the SEO related stuff [aka Helmet](https://github.com/ieee-uottawa/ieeeuottawa.ca/pull/127/files#diff-273813ed821d3ea258d3f6ee7cf9dcca94b835d2f8319c30e5ba9d4d9169712eR70-R101)

Reply:
Resolved